R/language.R

```diff
@@ -33,10 +49,16 @@
 #' is_atomic_lang(quote(call()))
 #'
 #' # With the exception of names, atomic language objects have the
-#' # property of being self-quoting:
+#' # property of being self-quoting literals:
```

Comment:
Maybe we should make this the definition of an atomic language object? (i.e. `identical(x, quote(x))` is true whenever `x` is an atomic lang object)